vyper/venom/basicblock.py

```diff
@@ -399,15 +430,21 @@ def get_ast_source(self) -> Optional[IRnode]:
 
     def copy(self) -> IRInstruction:
         ret = IRInstruction(self.opcode, self.operands.copy(), self.output)
+        if self._extra_outputs:
```

Comment:
don't use truthy -- this is wrong in case `self._extra_outputs` has length 0, the pointer to the empty list will be shared with the original instruction.

i think it might be better to just use `copy.copy(self._extra_outputs)`